ctapipe/calib/camera/charge_extraction.py

```diff
@@ -0,0 +1,281 @@
+from abc import abstractmethod
+import numpy as np
+from traitlets import Int, Unicode
+
+from ctapipe.core import Component
+
+
+def all_subclasses(cls):
+    return cls.__subclasses__() + [g for s in cls.__subclasses__()
+                                   for g in all_subclasses(s)]
+
+
+class ChargeExtractor(Component):
```

Comment:
Adding metaclass=ABCMeta gives me an error I do not understand how to solve.

```
TypeError                                 Traceback (most recent call last)
/Users/Jason/Software/oxpytools/jason_testbench/calibration_tool.py in <module>()
      5 import numpy as np
      6
----> 7 from ctapipe.calib.camera.charge_extraction import *
      8
      9

/Users/Jason/Software/ctapipe/ctapipe/calib/camera/charge_extraction.py in <module>()
     11
     12
---> 13 class ChargeExtractor(Component, metaclass=ABCMeta):
     14     name = 'ChargeExtractor'
     15

TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
```

Do you know how to solve this?